packages/checkout/widgets-lib/src/widgets/sale/hooks/useSmartCheckout.ts

```diff
@@ -15,17 +15,17 @@ type UseSmartCheckoutInput = {
   checkout: Checkout | undefined;
   provider: Web3Provider | undefined;
   items: SaleItem[];
-  amount: string;
   tokenAddress: string;
 };
 
 export const useSmartCheckout = ({
   checkout,
   provider,
   items,
-  amount,
+  
   tokenAddress,
 }: UseSmartCheckoutInput) => {
+  const amount = '0';
```

Comment:
Please ignore this patch
The entire hook gets remove in next PR as is not longer required for multi currency flow.